src/main/java/com/iridium/iridiumskyblock/commands/BankGive.java

```diff
@@ -0,0 +1,70 @@
+package com.iridium.iridiumskyblock.commands;
```

Comment:
I guess it's not a good idea to have these in the normal commands package because it'd just be confusing. They behave differently to normal commands. Maybe put them in a subcommand or bank sub-package?